internal/configs/configmaps.go

```diff
@@ -499,6 +499,15 @@ func ParseConfigMap(cfgm *v1.ConfigMap, nginxPlus bool, hasAppProtect bool, hasA
 				glog.Error("ConfigMap Key 'app-protect-physical-memory-thresholds' must follow pattern: 'high=<0 - 100> low=<0 - 100>'. Ignoring.")
 			}
 		}
+		if appProtectReconnectPeriod, exists := cfgm.Data["app-protect-reconnect-period-seconds"]; exists {
+			if period, err := ParseFloat64(appProtectReconnectPeriod); err == nil {
+				if period > 0 && period <= 60 {
```

Comment:
@galitskiy do we want to throw an error if the value is not between the accepted values? I think it would be easier to catch if we output something when the value is incorrect